src/lib/classes/lsp3-universal-profile.spec.ts

```diff
@@ -1,27 +1,27 @@
-import crypto from 'crypto';
-
 import { ERC725 } from '@erc725/erc725.js';
 import KeyManagerContract from '@lukso/universalprofile-smart-contracts/artifacts/LSP6KeyManager.json';
 import UniversalProfileContract from '@lukso/universalprofile-smart-contracts/artifacts/UniversalProfile.json';
 import { providers } from 'ethers';
-import { ethers, SignerWithAddress } from 'hardhat';
+import { ethers } from 'hardhat';
 import { Observable } from 'rxjs';
 
 import { UniversalProfile__factory } from '../../../build/main/src';
 import { LSPFactory } from '../../../build/main/src/lib/lsp-factory';
+import { testSetData, testUPDeploymentWithBaseContractFlag } from '../../../test/test.utils';
 import {
   ADDRESS_PERMISSIONS_ARRAY_KEY,
   DEFAULT_PERMISSIONS,
   PREFIX_PERMISSIONS,
 } from '../helpers/config.helper';
 
 import { lsp3ProfileJson } from './../../../test/lsp3-profile.mock';
-import { DeploymentEvent } from './../interfaces';
+import { DeployedContracts, DeploymentEvent } from './../interfaces';
+import { ProxyDeployer } from './proxy-deployer';
 
 jest.setTimeout(60000);
 jest.useRealTimers();
 describe('LSP3UniversalProfile', () => {
-  let signers: SignerWithAddress[];
+  let signers: any[];
```

Comment:
Is there absolutely no way to type this?

Reply:
There is some issue with the hardhat typing, the `SignerWithAddress` import was broken so this typing wasn't actually doing anything so just removed it. Will look to find a better solution than `any`

Reply:
@CallumGrindle 

See if importing the `SignerWithAddress` type from `@nomiclabs/hardhat-ethers` fix it. Like this:

```typescript
import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers";
```

Maybe this will solve the issue?
I think there was a similar issue in the tests of the `@lukso/lsp-smart-contracts`. And it's how it got fixed: https://github.com/lukso-network/lsp-smart-contracts/blob/ee34f45457cf250b7046bfb7ae3ade9767e2764f/tests/LSP6KeyManager/LSP6KeyManager.spec.ts#L1

Reply:
Does the trick. Thanks :)